bool enableAltitudeControl
bool enablePitchRollControl
bool enableYawControl
---
